pages/main.js

```diff
@@ -888,10 +911,25 @@ tryHomepageInit();
  * Handles the scroll animations
  */
 function scrollAnimation() {
+
+  /**
+   * 
+   * @param {HTMLElement} element 
+   * @returns {boolean}
+   */
+  function elementFilter(element){
+    let ret = true;
+    ret = ret && element
+    ret = ret && !element.classList.contains('nofade');
```

Comment:
Style nitpick - this code base avoids single tick quotes for strings whenever possible.